projects/imagemagick/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+# Copyright 2017 Google Inc.  #
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+################################################################################
+
+# move the corpus
+mkdir afl_testcases
+(cd afl_testcases; tar xvf "$SRC/afl_testcases.tgz")
+mkdir corpus
+find afl_testcases -type f -exec mv -n {} corpus/ \;
+zip -rj afl_testcases.zip corpus/
+
+./configure --prefix="$WORK" --disable-shared --disable-docs
+make "-j$(nproc)"
+make install
+
+$CXX $CXXFLAGS -std=c++11 -I"$WORK/include/ImageMagick-7" "$SRC/encoder_list.cc" \
+    -o "$WORK/encoder_list" \
+    -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 \
+    "$WORK/lib/libMagick++-7.Q16HDRI.a" \
```

Comment:
just to make sure: `7.Q16`is not going to change? Might it be better to use something like `libMagick++*.a` instead?

Reply:
7.Q16HDRI will change if/when the ImageMagick project does the following:
* Changes the default quantum depth
* Changes the default for HDRI being enabled
* Upgrades the major version

In any of those scenarios we're likely to need to make additional changes to the fuzzers to accommodate API changes and altering the `DMAGICKCORE_HDRI_ENABLE` and `DMAGICKCORE_QUANTUM_DEPTH` compiler variables.

Reply:
Makes sense, thanks!